next/components/forms/simple-components/MLink.tsx

```diff
@@ -24,7 +24,7 @@ const MLink = forwardRef<HTMLAnchorElement, LinkProps>(
     const plausible = usePlausible()
 
     const styles = twMerge(
-      cx('underline-offset-2', {
```

Comment:
Why this change?

Reply:
that was to allign offset of Button link variant and MLink underline. But after I've looked trough it, it's better to set it in Button variants then to remove it from MLink